src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String PERIOD_FOR = "Report for period ";
+    private static final String DATE_FORMAT = "dd.MM.yyyy";
+    private static final String SEPARATOR = " - ";
+    private static final String REGEX = " ";
+    private static final int INDEX_DAY = 0;
+    private static final int INDEX_HOURS_PER_DAY = 2;
+    private static final int INDEX_SALARY_PER_HOUR = 3;
+    private LocalDate localDateFrom;
+    private LocalDate localDate;
+    private LocalDate localDateTo;
+    private DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
+    private StringBuilder salaryInfo;
+
+    private int[] salary;
```

Comment:
All this fields should be local variables in the method

```suggestion
    private DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
```